src/main/java/org/skriptlang/skript/lang/entry/ContainerEntryData.java

```diff
@@ -0,0 +1,59 @@
+package org.skriptlang.skript.lang.entry;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.config.SectionNode;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.entry.EntryValidator.EntryValidatorBuilder;
+import org.skriptlang.skript.lang.structure.Structure;
+
+/**
+ * Virtually the same as {@link EntryValidator}.
+ * Allows an {@link EntryValidator} used within a {@link Structure} to contain embedded {@link EntryValidator}s.
```

Comment:
```suggestion

/**
 * An entry data for handling a {@link SectionNode} as the root node of another {@link EntryValidator}. This enables nested entry data validation.
```
This might be a bit clearer?